### PR TITLE
msmtp: install docs and examples

### PR DESCRIPTION
**Pull request by flokli**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/networking/msmtp/default.nix [hunk 1]
```diff
@@ -30,6 +31,10 @@ in stdenv.mkDerivation rec {
     stdenv.lib.optional stdenv.isDarwin [ "--with-macosx-keyring" ];
 
   postInstall = ''
+    install -d $out/share/doc/${pname}
+    install -Dm644 doc/*.example $out/share/doc/${pname}
+    cp -r scripts/{find_alias,msmtpqueue,msmtpq,set_sendmail} $out/share/doc/${pname}
```
Comment (grahamc) [inline pkgs/applications/networking/msmtp/default.nix]: Should we copy in to a scripts dir?

**Comment by grahamc** [comment]
@GrahamcOfBorg build msmtp

File: pkgs/applications/networking/msmtp/default.nix (hunk 1 above)
Comment (flokli) [reply pkgs/applications/networking/msmtp/default.nix]: Just `$out/scripts`? Fine for me too…

**Comment by grahamc** [comment]
@GrahamcOfBorg build msmtp

**Comment by flokli** [comment]
will rebase

**Comment by grahamc** [comment]
I think this is a bug in the builder, not an issue with rebasing :(

**Comment by flokli** [comment]
Went through successful now. Whatever ;-)

**Pull request merged**